src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
```

Comment:
Think about upper limit of ball number. Please see [this article](https://www.geeksforgeeks.org/java-util-random-nextint-java/) to learn about upper bound of `random.nextInt()`